streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1086,6 +1088,34 @@ private boolean nodeGroupContainsGlobalSourceNode(final Set<String> allNodesOfGr
         return false;
     }
 
+    private static class NodeComparator implements Comparator<org.apache.kafka.streams.TopologyDescription.Node>, Serializable {
```

Comment:
Should we import `org.apache.kafka.streams.TopologyDescription.Node` to shorten code here?

Reply:
ack.